content/docs/command-reference/plots/index.md

```diff
@@ -12,46 +12,37 @@ usage: dvc plots [-h] [-q | -v] {show,diff,modify} ...
 
 positional arguments:
   COMMAND
-    show         Generate plot from a metrics file.
-    diff         Plot differences in metrics between commits.
-    modify       Modify display properties of data-series plots (has no effect on image-type plots).
+    show         Generate plots from target files or plots definitions from `dvc.yaml` file.
+    diff         Show multiple versions of plot data by plotting them in a single image.
+    modify       Modify display properties of data-series plot outputs (has no effect on image-type plots).
 ```
 
-## Types of metrics
-
-DVC has two concepts for metrics, that represent different results of machine
-learning training or data processing:
```

Comment:
Should we also kill or rephrase this section in the `metrics` ref. since we're now explaining `plots` as a data-related command? As well as other `plots` links from that ref.

See https://dvc.org/doc/command-reference/metrics#types-of-metrics and 3rd paragraph in the **Description** below that.

Reply:
I agree. Should we just remove it? It seems to me that it doesn't make sense to keep this paragraph.

Reply:
> Should we just remove it?

Yes please @pared